### DISCUSSION TITLE
New User - Exported clips play on original MacBook but only audio on other devices

### DISCUSSION THREAD

**Discussion by kdammann** [mifi/lossless-cut#1247] [Support]
We exported clips from our .mov files on a MacBook (newer machine with M1 chip) into many .mov files. The clips look great on the original computer, but when we copy or send to another MacBook or phone, the clips are .mov still but audio-only, and we get the message: This file contains some media which isn't compatible with Quicktime player. 

I searched for this issue in Issues and Discussions but didn't see it. Thanks in advance for help!

**Comment by mifi** [comment]
I think the problem could be related to AirDrop. Does the problem happen if not using AirDrop?
Another thing to try is to disable any extra tracks other than 1 audio and 1 video track and see if that helps. If you could provide a report from the help menu that would also be very helpful

**Comment by kdammann** [comment]
Hi Mikael,

Apologies for delay, we were out of town but back and ready to
troubleshoot. :) I really appreciate your following up and trying to help
us figure this out - thank you!!

I'm copying my husband Mark, who is going to run some tests using your
suggestions. We will send reports from the help menu..

Thank you!
Kathleen

**Comment by kdammann** [comment]
Hi Mikael:

Mark Dammann here, Kathleen’s husband.

Thanks so much for troubleshooting this problem.

Turns out, after many hours researching the problem including 90 minutes on the phone with Applecare, it seems the problem has nothing to do with AirDrop (or with Lossless Cut). Rather, it seems to be a Quicktime problem. In two of our four Macs, the video codec with which the .mov file was encoded in 2012 (Apple Intermediate Codec) can not be read by 
QuickTime. (Strangely, our oldest and newest machines can read it fine, and the two middle machines can’t.) This despite identical versions of OS and Quicktime on all the machines. Very strange.

The solution, though not perfect, is to make a copy of the .mov file and convert the video codec from AIC to H.264, which can be read universally. (I have not yet used Lossless Cut with a .mov file that has been converted in this manner, but I’m confident it will work since Quicktime can open those files.) Unfortunately, this is not a lossless process, but in a test the new video looks quite similar to the original. An alternative is apparently to convert the video codec to Pro Res, which is lossless, but the Applecare representative said the file would be huge, perhaps 10X the original. Plus, we would still need to export in H.264 or some other format, which would presumably not be lossless.

Thanks again. It was very kind of you to take the time to provide assistance when Kathleen reached out. We are looking forward to returning to our project using your software, which is very well designed.

Best,

Mark & Kathleen

**Reply by mifi** [reply]
Hi!
It could possibly be a bug in ffmpeg. I have not done any testing with AIC (TBH I never even heard of the codec until now). After Kathleen initially had the problem I've released a new version so if you make sure that you're on the latest version of LosslessCut and try again, it will be using ffmpeg v5 (instead of the old v4). That might be worth trying, in case the ffmpeg team did some improvements on AIC format support. Other than that unfortunately I'm lost and don't really have any solutions, because it's so hard for me to reproduce the issue (as you say only reproducible on your "middle" machine 😅)
